packages/select/src/common/listItemsProps.ts

```diff
@@ -12,12 +12,21 @@ import { ItemListPredicate, ItemPredicate } from "./predicate";
 /** Reusable generic props for a component that operates on a filterable, selectable list of `items`. */
 export interface IListItemsProps<T> extends IProps {
     /**
-     * The currently focused item for keyboard interactions, or `null` to
+     * The currently focused item for keyboard interactions, or `undefined` to
      * indicate that no item is active. If omitted, this prop will be
      * uncontrolled (managed by the component's state). Use `onActiveItemChange`
      * to listen for updates.
      */
-    activeItem?: T | null;
+    activeItem?: T;
```

Comment:
bring back `| null`. we should clarify docs that `undefined` means uncontrolled.

Reply:
While reverting, I'm updating documentation of relevant props (like this one) to say "If omitted or `undefined`, this prop will be uncontrolled"

Reply:
Done. Is there any other documentation that should be updated in relation to this?